sdcm/utils/nemesis_utils/node_operations.py

```diff
@@ -0,0 +1,57 @@
+import contextlib
+import logging
+
+from typing import Optional
+from random import choice
+
+from sdcm.cluster import BaseNode
+
+LOGGER = logging.getLogger(__name__)
+
+
+@contextlib.contextmanager
+def block_scylla_ports(target_node: "BaseNode", ports: list[int] | None = None):
+    ports = ports or [7001, 7000, 9042, 9142, 19042, 19142]
+    target_node.install_package("iptables")
+    target_node.log.debug("Block connections %s", target_node.name)
+    for port in ports:
+        target_node.remoter.sudo(f"iptables -A INPUT -p tcp --dport {port} -j DROP")
+        target_node.remoter.sudo(f"iptables -A OUTPUT -p tcp --dport {port} -j DROP")
+    yield
+    target_node.log.debug("Remove all iptable rules %s", target_node.name)
+    for port in ports:
+        target_node.remoter.sudo(f"iptables -D INPUT -p tcp --dport {port} -j DROP")
+        target_node.remoter.sudo(f"iptables -D OUTPUT -p tcp --dport {port} -j DROP")
+
+
+@contextlib.contextmanager
+def stop_scylla_with_sigstop(target_node: "BaseNode"):
+    target_node.log.debug("Send signal SIGSTOP to scylla process on node %s", target_node.name)
+    target_node.remoter.sudo("pkill --signal SIGSTOP -e scylla", timeout=60)
+    yield
+    target_node.log.debug("Send signal SIGCONT to scylla process on node %s", target_node.name)
+    target_node.remoter.sudo(cmd="pkill --signal SIGCONT -e scylla", timeout=60)
+
+
+def get_random_free_verification_node(node: "BaseNode") -> "BaseNode":
```

Comment:
There is a function for this that marks the node as nemesis, no need to reinvent it

This isn't safe for parallel nemesis

Reply:
i removed this function and use exist one